Source/Core/CircleGeometry.js

```diff
@@ -174,6 +176,17 @@ define([
             vertexFormat : VertexFormat.POSITION_ONLY
         });
     };
+    
+    defineProperties(CircleGeometry.prototype, {
+        /**
+         * @private
+         */
+        rectangle : {
```

Comment:
Throughout, this is probably better named `boundingRectangle`.  At first, I was confused, for example, why a circle returns a rectangle.

The `bounding` prefix is also used throughout the graphics code.